chat.js

```diff
@@ -137,7 +137,7 @@ class PatternTester {
 	register(...elems) {
 		for (let elem of elems) {
 			this.elements.push(elem);
-			if (/^[^ \^\$\?\|\(\)\[\]]+ $/.test(elem)) {
+			if (/^[^ ^$?|()[\]]+ $/.test(elem)) {
```

Comment:
Does it actually complain about `\[`? Ugh. I'd put it back unless it forced me to take it out.